casper/src/test/scala/coop/rchain/casper/helper/TestNode.scala

```diff
@@ -195,25 +183,12 @@ case class TestNode[F[_]: Concurrent: Timer](
   // This method assumes that block will be created successfully
   def createBlockUnsafe(deployDatums: Signed[DeployData]*): F[BlockMessage] =
     for {
-      _                 <- deployDatums.toList.traverse(MultiParentCasper.deploy[F])
-      preState          <- MultiParentCasper.getPreStateForNewBlock[F]
-      preState          <- MultiParentCasper.getPreStateForNewBlock[F]
```

Comment:
There was duplicated lines of code. I guess it was just mistake so I removed one of them.